mlflow/store/sqlalchemy_store.py

```diff
@@ -40,7 +40,7 @@ def __init__(self, db_uri, default_artifact_root):
         """
         Create a database backed store.
 
-        :param db_uri: SQL connection string used by SQLAlchemy Engine to connected to database.
+        :param db_uri: SQL connection string used by SQLAlchemy Engine to connect to database.
```

Comment:
SQL connection string used by SQLAlchemy Engine to connect to database. -> SQL connection string used by SQLAlchemy Engine to connect to the database.